run.sh

```diff
@@ -233,9 +233,9 @@ fi
 # fix for broken steamcmd app_info_print: execute install/update manually, checking for updates fails.
 # https://github.com/ValveSoftware/steam-for-linux/issues/9683#issuecomment-1826928761
 if [ ! -f "$ARKSERVER/steamapps/appmanifest_376030.acf" ]; then
-  arkmanager install
+  arkmanager install ${VERBOSE:+--verbose}
```

Comment:
I'd additionally suggest to define the variable at the very beginning, like the others with `:-` and a default value.

That hasn't been followed consistently, so I see how that seems to be completely random. For me it's nice to see the variables used at the beginning of a script in general.

Reply:
done